scripts/cv_api_master.py

```diff
@@ -254,9 +254,102 @@
 #     ],
 # }
 
-MODELS_MASTER = {
-    "path": "models/",
-    "title": "Models",
+FUNCTIONS_MASTER = {
```

Comment:
Shouldn't these be moved to the backbones section with consistent structure? These are from the old API originally

Reply:
These cannot be used the same way as the `Backbone`s, right? Put them in the same section with `Backbone`s may be confusing.

Reply:
I'm saying that these models have been ported to `Backbone` and you are documenting the old API here. Let's move these models to the `Backbone` section with consistent documentation

Reply:
Done.